docker/utils/rotate-new-api-keys.sh

```
@@ -0,0 +1,90 @@
+fi
+
+if [ "$update_env" != "true" ]; then
+    exit 0
+fi
+
+echo "Updating .env..."
+
+for var in SUPABASE_PUBLISHABLE_KEY SUPABASE_SECRET_KEY; do
+    eval "val=\$$var"
+    if grep -q "^${var}=" .env; then
+        sed -i.old -e "s|^${var}=.*$|${var}=${val}|" .env
```

Shellcheck (0.11.0)

[warning] val is referenced but not assigned (for output from commands, use "$(val ...)" ).

(SC2154)